libs/feature/search/src/lib/organisations/organisations.container.component.html

```diff
@@ -0,0 +1,3 @@
+<gn-ui-organisations-list
```

Comment:
not sure that this component is usefull.

Reply:
Yes, I added it to display something. So if I get it right, the idea is to use the organisation cards and pagination dumb compnents directly in the smart component and not add an intermediate dumb component for listing.